Assignment2/src/serviceUtil/inputOutputUtil.java

```diff
@@ -0,0 +1,96 @@
+package serviceUtil;
+
+import model.Course;
+import model.Student;
+
+
+import java.util.ArrayList;
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.Scanner;
+import util.fileReadWriteUtil;
+public class inputOutputUtil {
+
+    public static Student getInputUserDetails() {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the following details:");
+        System.out.print("Name: ");
+        String name = sc.next();
+        System.out.print("Age: ");
+        int age = inputAge();
+        System.out.print("Address: ");
+        String address = sc.next();
+        // Roll Number
+        int rno = inputRno();
+
+        // Courses
+        int nCourses = inputCourses();
+        ArrayList<String> courses = new ArrayList<>();
+        System.out.print("Courses Undertaken(A,B,C,D,E,F): ");
+        for (int i = 0; i < nCourses; i++) {
+            courses.add(sc.next());
+        }
+        // set and return details of Student and Course objects
+        return new Student(rno, name, age, address, new Course(courses));
+    }
+    private static int inputCourses() {
```

Comment:
Should this static method be allowed to access without using any instance of Student ???